src/Makefile

```diff
@@ -11,7 +11,7 @@ include $(CURRENT_VERSION_FILE)
 export VDO_VERSION
 
 SUBDIRS                 := perl c++ python tools
-CLEAN_SUBDIRS           := $(SUBDIRS) stats packaging
+CLEAN_SUBDIRS           := $(SUBDIRS) stats packaging packaging/dmlinux
 CHECK_SUBDIRS           := perl tools python/vdo
```

Comment:
Please don't skip levels like this. Instead, update the Makefile in src/packaging to clean the dmlinux directory.

Reply:
I don't think we should have clean and all from packaging contain dmlinux. dmlinux is very separate from the default make stuff and shouldnt add time to those targets for what is a very specific thing. When I decided this, I meant to remove packaging/dmlinux from CLEAN_SUBDIRS but forgot. This is especially true for all since dmlinux all requires DMLINUX_SOURCE_DIR to be set. If we want to add dmlinux just to the clean part of packaging I could maybe get behind that, but I think having clean all in the make target dmlinux-jenkins-parallel itself makes the most sense

Reply:
Just removing packaging/dmlinux from this list seems fine.